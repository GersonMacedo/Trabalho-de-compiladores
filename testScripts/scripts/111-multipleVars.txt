! -s SYNTACTIC
! should accept multiple vars
program p ;
    var a : integer ;
    var b : boolean ;
    var c : boolean ;
    var d : integer ;
    var e : integer ;
    var f : boolean ;
begin
end .