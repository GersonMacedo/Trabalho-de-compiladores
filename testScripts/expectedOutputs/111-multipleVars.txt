
Performing up to syntactic analysis at testScripts/scripts/111-multipleVars.txt
No errors found
